docs/api.md

```diff
@@ -3,143 +3,153 @@ layout: default
 title: api
 nav_order: 5
 ---
+
```

Comment:
This file should not be edited manually as it is automatically generated based on the documentation written in the library js file using jsdoc-to-markdown.

Please document functions here https://github.com/id-studiolab/MLTK01/blob/master/library/mltk.js and regenerate the api.md by executing gulp generate-doc

Reply:
Okok; I think VSCode linted the page when I opened it. I'll use the gulp command to regenerate

Reply:
![Screenshot 2023-02-16 at 11 12 19](https://user-images.githubusercontent.com/7902129/219335867-c1d2931e-ab15-426c-9443-8a11a3cd8dd2.png)

So I'm awful at NPM/Node things right now. I installed gulp and tried running the command and got the following error.

Reply:
I think there is a bug in the library that keeps giving me this error as well... the only workaround I found so far is to enter the gulp file here https://github.com/id-studiolab/MLTK01/blob/0f42b1938f168c3e84a4fcc125f1ac02f833f199/gulpfile.js#L12-L13

- Uncomment one line 12 and comment line 13, save the file and run the gulp generate doc command. 
- Then do the opposite, resave and run the command again.

It seems like somehow the template file get cached or something and the second time you run it with the same content doesn't work

Reply:
actually, fuck gulp! 
I just pushed some modification to master to be able to generate the documentation with a npm task... if you pull master you should be able to run npm run docs and the api documentation should automatically update.